cmd/periph-smoketest/gpiosmoketest/gpiosmoketest.go

```diff
@@ -217,69 +229,74 @@ func (s *SmokeTest) testEdgesBoth(p1, p2 gpio.PinIO) error {
 		return err
 	}
 	time.Sleep(s.shortDelay)
-	if c := s.waitForEdge(p1); <-c {
-		// The linux kernel makes it hard to enforce this. :(
+	if !<-s.waitForEdge(p1, false) {
```

Comment:
Having wantEdge and wantNoEdge (or similar) consts would make this a little clearer.

Reply:
Replaced waitForEdge with expectEdge() and expectNoEdge() instead.